microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-   ![An Audits report after enabling text compression](../media/speed-glitch-tony-remix-updated-audits-performance.msft.png)
+1. In the **Coverage** tool, click **Start instrumenting coverage and refresh the page** (![Refresh icon](../media/reload-icon.msft.png)). The **Coverage** tool provides an overview of how much of the JavaScript and CSS code loaded on the page runs.
 
-Your overall performance score should have increased, meaning that the site is getting faster.
+    ![The Coverage tool, in the Drawer panel, showing the report of how much code is unused](./images/coverage-report.png)
 
-#### Text compression in the real world
+The coverage report confirms that the Bing Map dependencies contain code that's unused when the page loads. The map on the demo website isn't visible when the page first loads. Loading the Bing Map only when it becomes visible to users is a good opportunity to improve performance.
 
-Most servers have simple fixes like this for enabling compression.  Do a search on how to configure whatever server you use to compress text.
+Use the Intersection Observer API to detect when the map becomes visible to the user. The Intersection Observer API provides a way to observe changes in the intersection of a target element, the map in this case, with the webpage's viewport. To learn more, see [Intersection Observer API](https://developer.mozilla.org/docs/Web/API/Intersection_Observer_API) at MDN.
 
-### Resize images
+1. In Visual Studio Code, open the `/travel-site/index.html` file and scroll down to the bottom of the file. The Bing Map API is loaded by using a `<script>` tag:
 
-Your report indicates that avoiding enormous network payloads is one of the top opportunities for improving the performance of the page.  Resizing images helps reduce the size of the network payload.  If your user is viewing your images on a mobile device screen that is 500-pixels-wide, there is really no point in sending a 1500-pixel-wide image.  Ideally, you send a 500-pixel-wide image, at most.
+    ![VS Code, showing the index.html code and the Bing map script tag](./images/bing-map-script.png)
 
-1. In your report, select **Avoid enormous network payloads** to display which images should be resized.  It looks like two of the `.jpg` files are over 2000 KB, which is bigger than necessary.
+    Below this line is another line that's responsible for configuring and loading the map in the right place: `<script src="assets/map.js"></script>`
 
-   <!--
-   ![Details about the properly size images opportunity](../media/speed-glitch-tony-remix-updated-audits-performance-opportunities-expanded.msft.png)
-   -->
+1. Delete these two lines and add this new line instead: `<script src="assets/map-on-demand.js"></script>`.
 
-1. Back in the editor tab, open `src/model.js`.
+    ![VS Code, showing the index.html code and the new on-demand map script tag](./images/bing-map-on-demand-script.png)
```

Comment:
change dir name from /images/ to /get-started-images/ and put at same level as .md file.

I need to double check this to check my understanding... eg see if this dir already exists or is entirely new.  To see if focus-mode PR is affected.

Reply:
This directory already existed before. But was actually not used at all. So it's fine to rename it. I will do this now.